packages/beacon-node/src/network/peers/discover.ts

```diff
@@ -360,8 +429,11 @@ export class PeerDiscovery {
     peerId: PeerId,
     multiaddrTCP: Multiaddr,
     attnets: boolean[],
-    syncnets: boolean[]
+    syncnets: boolean[],
+    custodySubnetCount?: number
   ): DiscoveredPeerStatus {
+    const nodeId = computeNodeId(peerId);
+    this.logger.debug("handleDiscoveredPeer", {nodeId: toHexString(nodeId), peerId: peerId.toString()});
```

Comment:
should avoid using `toHexString` from ssz package as it's very slow

```suggestion
    this.logger.debug("handleDiscoveredPeer", {nodeId: toHex(nodeId), peerId: peerId.toString()});
```

Reply:
updated in #8163